WalletWasabi/CoinJoin/Client/Clients/CoinJoinClient.cs

```diff
@@ -93,7 +93,7 @@ public class CoinJoinClient
 
 	private ClientRoundRegistration? DelayedRoundRegistration { get; set; }
 
-	public Func<Uri> CcjHostUriAction { get; private set; }
+	public Func<Uri>? CcjHostUriAction { get; private set; }
```

Comment:
Are you sure we want this to be nullable? I don't think so.

The warning can be "eliminated" with a good old:
`CcjHostUriAction = Synchronizer.HttpClientFactory.BackendUriGetter ?? throw new Exception();
`